components/Button.tsx

```diff
@@ -0,0 +1,54 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+export const Button = ({
+  text,
+  defaultClass,
+  hoverClass,
+  href,
+  isExternal,
+}: {
+  text: string;
+  defaultClass: string;
+  hoverClass: string;
+  href: string;
+  isExternal: boolean;
+}) => {
+  const buttonClass =
```

Comment:
better to have such constants outside of the component, so there's no need to recreate this again each time the component renders